src/ui/components/Comments/TranscriptComments/NetworkRequestPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { setSelectedPanel } from "ui/actions/layout";
+import { showRequestDetails } from "ui/actions/network";
+import MaterialIcon from "ui/components/shared/MaterialIcon";
+import { getSummaryById } from "ui/reducers/network";
+import { UIState } from "ui/state";
+
+import { trackEvent } from "ui/utils/telemetry";
+
+export default function NetworkRequestPreview({ networkRequestId }: { networkRequestId: string }) {
+  const dispatch = useDispatch();
+  const request = useSelector((state: UIState) => getSummaryById(state, networkRequestId));
+
+  const onClick = () => {
```

Comment:
If the request has to be first loaded then there can/will be a delay. This is fine for now, but we should do a fast follow to save it in the DB